docs/user-guide/command-line-interface.md

```diff
@@ -315,6 +317,22 @@ Example:
 
     eslint -v
 
+#### `--disableInlineConfig`
+
+This option prevents inline comments like `/*eslint-disable*/` or
+`/*global foo*/` from having any effect. This allows you to set an ESLint
+config without files modifying it. All inline config comments are ignored, e.g.:
+
+* `/*eslint-disable*/`
+* `/*eslint-enable*/`
+* `/*global*/`
+* `/*eslint*/`
+* `/*eslint-env*/`
```

Comment:
Added the fix (it's in a separate commit but I can squash before merge, figured I'd wait until the end of review beforehand :smile:).

Thanks for catching that. :)